test/k8s/Node.go

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of Cilium
```

Comment:
```suggestion
// Copyright Authors of Cilium
```

Reply:
I thought you'd fixed it! :scream_cat:

Reply:
Crap :disappointed: 
I actually fixed it initially, but then I must have screwed up with git and lost that change

Reply:
Never mind :), I'll do another pass someday for the new files people were working on when we pushed the change.